sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/AbstractScheduler.java

```diff
@@ -43,14 +43,15 @@
 
     private Object inProgressLock = new Object();
     private Set<Protos.OfferID> offersInProgress = new HashSet<>();
+    private AtomicBoolean processingOffers = new AtomicBoolean(false);
 
     /**
      * Executor for handling TaskStatus updates in {@link #statusUpdate(SchedulerDriver, Protos.TaskStatus)}.
      */
     protected final ExecutorService statusExecutor = Executors.newSingleThreadExecutor();
 
     /**
-     * Executor for processing offers off the queue in {@code processOffers()}.
+     * Executor for processing offers off the queue in {@code executePlans()}.
```

Comment:
could do `{@link #executePlans()}` (or `{@link #executePlansLoop()}`) here, and then it should update automatically in any future IDE renames